build.gradle

```diff
@@ -297,14 +299,18 @@ task satoriTest(type: Test) {
 }
 
 task nakamaShadowJar(type: ShadowJar) {
+    configurations = [project.configurations.sharedImplementation,
```

Comment:
This allows the dependencies defined at the top of the file to get packed into the shadow jar properly.